File: src/main/java/com/tom/guess/Game.java
package com.tom.guess;

import java.util.Random;
import java.util.Scanner;

public class Game {
    public static void main(String[] args) {
        Random random = new Random();
        int secret = random.nextInt(10)+1;
        System.out.println(secret);
        Scanner sc = new Scanner(System.in);
        int num = 0;
        while (num != secret) {

            System.out.print("Please insert a number:");
            num = sc.nextInt();
            if(num > secret){
                System.out.println("lower");
            }else if(num < secret){
                System.out.println("higher");
            }else {
                System.out.println("bingo!!  the secret number is " + secret);
            }
        }
    }
}

File: src/main/kotlin/com/tom/guess/game.kt
package com.tom

import java.util.*

fun main(args: Array<String>) {
    val secret = Random().nextInt(10)+1
    println(secret)
//    val scanner = Scanner(System.`in`)
    var num = 0
    while (num != secret) {
        print("Please insert a number:")
//        num = scanner.nextInt()

        num = readLine()!!.toInt()  //kotlin readLine()可以類似scanner的用途
        if(num > secret){
            println("lower")
        }else if( num < secret){
            println("higher")
        }else{
            println("bingo!! the secret number is $secret")
        }

    }
}